cmd/gardenlet/app/migration.go

```diff
@@ -25,6 +29,11 @@ func (g *garden) runMigrations(ctx context.Context, log logr.Logger) error {
 		return err
 	}
 
+	log.Info("Deleting stale alertmanager VPAs")
+	if err := deleteStaleAlertmanagerVPAs(ctx, log, g.mgr.GetClient()); err != nil {
+		return err
+	}
```

Comment:
```suggestion
	return deleteStaleAlertmanagerVPAs(ctx, log, g.mgr.GetClient())
```

Reply:
I would rather prefer to keep it like it

The existing code is:
```go
	log.Info("Migrating deprecated failure-domain.beta.kubernetes.io labels to topology.kubernetes.io")
	if err := migrateDeprecatedTopologyLabels(ctx, log, g.mgr.GetClient(), g.mgr.GetConfig()); err != nil {
		return err
	}

	return nil
```

and not:
```go
	log.Info("Migrating deprecated failure-domain.beta.kubernetes.io labels to topology.kubernetes.io")
	return migrateDeprecatedTopologyLabels(ctx, log, g.mgr.GetClient(), g.mgr.GetConfig())
```

It makes adding new migration func calls easier.